src/Juvix/Compiler/Concrete/Translation/FromSource.hs

```diff
@@ -294,14 +297,21 @@ statement = P.label "<top level statement>" $ do
   optional_ stashPragmas
   ms <-
     optional
-      ( either StatementImport StatementOpenModule <$> importOpenSyntax
-          <|> StatementOpenModule <$> openModule
-          <|> StatementSyntax <$> syntaxDef
-          <|> StatementInductive <$> inductiveDef Nothing
-          <|> StatementModule <$> moduleDef
-          <|> StatementAxiom <$> axiomDef Nothing
+      ( either StatementImport StatementOpenModule
+          <$> importOpenSyntax
+          <|> StatementOpenModule
+            <$> openModule
+          <|> StatementSyntax
+            <$> syntaxDef
+          <|> StatementInductive
+            <$> inductiveDef Nothing
+          <|> StatementModule
+            <$> moduleDef
+          <|> StatementAxiom
+            <$> axiomDef Nothing
           <|> builtinStatement
-          <|> StatementFunctionDef <$> functionDefinition Nothing
+          <|> StatementFunctionDef
+            <$> functionDefinition Nothing
```

Comment:
Why these newlines here? I'd say they make things less readable.

Reply:
that was unintentional. If I run the formatter that comes with lsp these lines get introduced. I've reverted it